core/src/test/java/com/yahoo/oak/ConcurrentPutRemoveTest.java

```diff
@@ -100,22 +103,26 @@ public void testMain() throws InterruptedException {
             }
         }
 
-        for (int i = 0; i < NUM_THREADS; i++) {
-            threads.get(i).start();
-        }
-
         try {
             barrier.await();
         } catch (InterruptedException | BrokenBarrierException e) {
             e.printStackTrace();
+            Assert.fail("got unexpected exception");
         }
```

Comment:
If we propagate the exception here, I don't see a reason to catch it.
Just drop the try/catch clause.

Reply:
i did this